pyproject.toml

```diff
@@ -60,6 +60,7 @@ dependencies = [
   "python-multipart",  # see https://www.starlette.io/#dependencies
   "arize-phoenix-evals>=0.13.1",
   "fastapi",
+  "pydantic>=1.0,<3"
```

Comment:
isn't this implicit from fastapi?

Reply:
It is, but this PR makes Pydantic a direct (not just transitive) dependency.